FreeRTOS/FreeRTOS-Kernel
# Update github Auto-Release workflow to add an option to set the main branch version in task.h

Update github Auto-Release workflow to add an option to set the main branch version in task.h

Description
-----------
Adding text into this new field updates the following macros in task.h in the main branch:
tskKERNEL_VERSION_NUMBER (alphanumeric string)
tskKERNEL_VERSION_MAJOR (numeric only)
tskKERNEL_VERSION_MINOR (numeric only)
tskKERNEL_VERSION_BUILD (numeric only)

Test Steps
-----------
Tested on my personal forks of the FreeRTOS and FreeRTOS-Kernel repositories.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
